app/controllers/taxon_changes_controller.rb

```diff
@@ -357,6 +357,37 @@ def group
       format.html { render layout: "bootstrap" }
     end
   end
+
+  def analysis_with_url( row )
```

Comment:
This should be private. Also could use a more descriptive name. Maybe `identification_analysis_with_url` or `analyze_ids_row`